api/src/main/java/io/opencensus/metrics/NoopDoubleGaugeMetric.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.metrics;
+
+import io.opencensus.common.ToDoubleFunction;
+import io.opencensus.internal.Utils;
+import java.util.List;
+
+/** No-op implementations of DoubleGaugeMetric class. */
+final class NoopDoubleGaugeMetric extends DoubleGaugeMetric {
```

Comment:
This can be moved to the DoubleGaugeMetric, that way is consistent with other no-ops. Same for the Long version.